.github/workflows/ci_pipeline.yml

```diff
@@ -1,4 +1,4 @@
-name: CI pipeline
+name: CI pipeline (Colcon)
```

Comment:
@nahueespinosa nit: may as well call this `ci_pipeline_colcon.yml` or even `colcon_ci_pipeline.yml`.

Reply:
Done in https://github.com/Ekumen-OS/beluga/pull/521/commits/2c5cfdca94021142a277e7e4ae6a5aa64f5658b7. I also added a new badge.

Reply:
@marcoshuck https://github.com/Ekumen-OS/beluga/pull/521/commits/49b0b13fa1c51c25cd26b07b35707142223a1dd6.